lib/text_config.dart

```diff
@@ -52,10 +53,13 @@ class TextConfig {
   /// For proper fonts of languages like Hebrew or Arabic, replace this with [TextDirection.rtl].
   final TextDirection textDirection;
 
+  final MemoryCache _textPainterCache =
+      MemoryCache<String, material.TextPainter>();
+
   /// Creates a constant [TextConfig] with sensible defaults.
   ///
   /// Every parameter can be specified.
-  const TextConfig({
+  TextConfig({
```

Comment:
How come they can't be const anymore?

Reply:
Now `_textPainterCache` contains some non const collections.

Reply:
Exactly!